js_modules/dagster-ui/packages/ui-components/src/components/BaseButton.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import {Colors} from './Color';
 import {StyledButton, StyledButtonText} from './StyledButton';
 
-interface CommonButtonProps {
+export interface CommonButtonProps {
```

Comment:
exporting this one is necessary because AnchorButton uses `React.ComponentPropsWithRef<'a'>`